userdocs/profiles.dev/docs/catalog-docs/add-profiles.md

```diff
@@ -0,0 +1,180 @@
+---
+sidebar_position: 1
+---
+
+# Adding profiles
+
+:::info
+Please refer to the [Introduction](/docs/intro#catalog) for a list of terms
+around Catalogs.
+:::
+
+To add profiles to a catalog, you must first deploy the catalog API
+and `ProfileCatalogSource` controller to your cluster.
+
+This can be done with the `pctl prepare` command. By default, this command
+will install latest, but you can pass a specific version with the `--version`
+flag.
+
+Once deployed, there are two methods to add profiles to the cluster catalog:
+dynamically and manually.
+
+## Dynamically populate a catalog
+
+To dynamically add all profiles and tags from profiles repositories to the catalog,
+create a `ProfileCatalogSource` providing the repositories' URLs,
+and apply the resource to the cluster.
+
+For example, the following manifest will discover all the profiles within the
+[weaveworks/profiles-examples](https://github.com/weaveworks/profiles-examples) repo
+and add an entry in the catalog for each profile tag.
+
+```yaml
+apiVersion: weave.works/v1alpha1
+kind: ProfileCatalogSource
+metadata:
+  name: nginx-catalog
+spec:
+  repositories:
+  - url: https://github.com/weaveworks/profiles-examples
+```
+
+After applying the manifest and waiting a moment, we can use `pctl search` to see the
+catalogued profiles:
+
+```bash
+$ kubectl apply -f dynamic-catalog-source.yaml
+# allow a few moments. the more profiles/tags, the more time the catalog manager
+# will need to discover them all
+
+$ pctl search --all
+CATALOG/PROFILE                 VERSION DESCRIPTION
+nginx-catalog/bitnami-nginx     v0.0.2  Profile for deploying local nginx chart
+nginx-catalog/weaveworks-nginx  v0.1.0  Profile for deploying nginx
+nginx-catalog/bitnami-nginx     v0.0.1  Profile for deploying local nginx chart
+nginx-catalog/weaveworks-nginx  v0.1.1  Profile for deploying nginx
+```
+
+Once added, the catalog will monitor each profile and update the catalog entries
+when new versions are released.
+
+### Adding profiles from private repositories
+
+To dynamically add profiles from a private repository, you must provide a reference to a
+secret in your catalog source manifest.
+
+Create a [Secret](https://kubernetes.io/docs/concepts/configuration/secret/)
+to hold your git provider access credentials.
+- [HTTPS repositories](#https-repositories)
+- [SSH repositories](#ssh-repositories)
+
+
+And add the name to the relevant repo in the `spec`:
+
+```yaml
+apiVersion: weave.works/v1alpha1
+kind: ProfileCatalogSource
+metadata:
+  name: nginx-catalog
+spec:
+  repositories:
+  - url: https://github.com/weaveworks/profiles-examples
+    secretRef:
+      name: <name of secret>
+```
+
+#### HTTPS repositories
```

Comment:
```suggestion
#### HTTPS repositories

```

Reply:
what's the diff here?

Reply:
Plus one newline.

Reply:
the markdown will still render, but sure 😄